### ISSUE TITLE
Can't autorize specific file type

### ISSUE THREAD

**Issue by troublestarter** [MailCleaner/MailCleaner#512]
Hi,

You can also see below : https://forum.mailcleaner.org/viewtopic.php?f=2&t=3357&p=13680#p13680

We can't add filetype and ask to whitelist them.
I tried to add the filetype and disable it but it's not whitelisting it.

Can you specify how to do this ?

Thanks

**Comment by JohnMertz** [comment]
Do you have the file /var/mailcleaner/spool/mailcleaner/mc-experimental-macros on your machine? If you aren't specifically blocking .xlsb (Configuration->Content protection->Attachment type), it is likely being blocked because of the macro protection in Exim.

Otherwise, you would need to include the log line(s) where the file gets rejected for us to understand what process is doing it.

Also note that if there is a specific sender you would like to allow these from, you can just create a 'full whitelist' for that sender/domain which will bypass even virus filtering: https://support.mailcleaner.net/boards/3/topics/95-full-whitelist

**Comment by troublestarter** [comment]
Hello @JohnMertz 

No : ls: cannot access /var/mailcleaner/spool/mailcleaner/mc-experimental-macros: No such file or directory

I will look at the full whitelist to test it.

On this link : https://support.mailcleaner.net/boards/3/topics/35-news-about-virus-waves
It is explained that docm and other extensions has been explicitely blacklist at smtp stage.

_2016/05/26 11:00:00
Due to a recent virus wave containing infected .docm, .dotm, .wsf and .js (in zip, archive) , we deployed a fix blocking all .docm, .dotm, .wsf and .js both as attachments and archives.
This is currently the best solution we found. This can be disabled by simply deleting a file on each of your nodes :

rm /var/mailcleaner/spool/mailcleaner/mc-experimental-docm
rm /var/mailcleaner/spool/mailcleaner/mc-experimental-jsinzip
rm /var/mailcleaner/spool/mailcleaner/mc-experimental-wsf_

Can be a solution ?
